ui/app/AppLayouts/Communities/panels/SharedAddressesPermissionsPanel.qml

```diff
@@ -424,7 +489,13 @@ Rectangle {
                                             }
                                         }
 
-                                        color: overallPermissionRow2.tokenCriteriaMet ? Theme.palette.directColor1 : Theme.palette.baseColor1
+                                        color: {
+                                            if (channelPermsSubPanel.permissionLost)
+                                                return Theme.palette.dangerColor1
+                                            if (overallPermissionRow2.tokenCriteriaMet)
+                                                return Theme.palette.directColor1
+                                            return Theme.palette.baseColor1
```

Comment:
This color snippet is used 3 times it seems, so maybe you could put it in a shared function?

Reply:
Well actually they are not the same colors :/ The "success" text color is `Theme.palette.directColor1` while the icon color is `Theme.palette.successColor1`

Reply:
Oh my bad